types/ext/translation-internal.d.ts

```diff
@@ -40,15 +40,29 @@ export type TextDeinflectionOptionsArrays = [
     emphatic: [collapseEmphatic: boolean, collapseEmphaticFull: boolean][],
 ];
 
+export type TermDictionaryEntryInternal = Omit<Dictionary.TermDictionaryEntry, 'inflectionRuleChainCandidates'> & {
```

Comment:
The `Internal` suffix to these names is redundant since they are in the `translation-internal` file, but may help to differentiate  them from their `dictionary.d.ts` counterparts. Keep or remove?

Reply:
I never understood what "internal" semantically meant. It feels weird to use types to signal whether something is a private or public structure